selenium/src/test/java/au/org/emii/portal/tests/step1/StepsDisabledTest.java

```diff
@@ -0,0 +1,40 @@
+package au.org.emii.portal.tests.step1;
+
+import au.org.emii.portal.tests.BaseTest;
+import au.org.emii.portal.tests.step3.ShapeFileTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.apache.log4j.Logger;
+
+import java.util.List;
+
+public class StepsDisabledTest extends BaseTest {
+    private static Logger log = Logger.getLogger(ShapeFileTest.class.getName());
+
+    @Test
+    public void stepsDisabledTest() throws InterruptedException{
+        WebDriver driver = getDriver();
+        driver.get(AODN_PORTAL_SEARCH_PAGE);
+
+        List<WebElement> buttons = webElementUtil.findElements(By.className("viewPortButton"));
+
+        buttons.get(1).click();
```

Comment:
Would it be better to  match on button text here to make it clearer what button is being selected?  I had to go to the portal and use the debugger to work out exactly what button was being pressed.